stages/org.osbuild.ostree.commit

```diff
@@ -43,6 +43,11 @@ SCHEMA_2 = """
     "parent": {
       "description": "commit id of the parent commit",
       "type": "string"
+    },
+    "selinux-label-version": {
+      "description": "Set selinux label version",
+      "type": "integer",
```

Comment:
Out of curiosity can this schema language constrain to 0 | 1 right now?

Reply:
We could constrain it to a range: https://json-schema.org/understanding-json-schema/reference/numeric#range though its behavior seems to have changed over the json schema specifications.

Reply:
Eh, not really worth it